remoteappmanager/rest/registry.py

```diff
@@ -0,0 +1,52 @@
+from remoteappmanager.rest.resource import Resource
+
+
+class Registry:
+    def __init__(self):
+        self._registered_types = {}
+
+    def register(self, typ):
+        """Registers a Resource type with an appropriate
```

Comment:
Why use an under variable (__collection_name__) instead of passing collection name as an argument in this method?